tests/integration/ceph_flex_test.go

```diff
@@ -167,9 +168,13 @@ func (s *CephFlexDriverSuite) statefulSetDataCleanup(poolName, storageClassName,
 	listOpts := metav1.ListOptions{LabelSelector: "app=" + statefulSetName}
 	// Delete stateful set
 	err := s.kh.Clientset.CoreV1().Services(defaultNamespace).Delete(statefulSetName, &delOpts)
-	assert.NoError(s.T(), err)
+	if !errors.IsNotFound(err) {
```

Comment:
Can the changes in this file also call `assertNoErrorUnlessNotFound(s, err)`?

Reply:
Done.